The following modules were not unloaded:
  (Use "module --force purge" to unload all):

  1) modenv/scs5
Module Python/3.6.4-intel-2018a and 18 dependencies loaded.
Module cuDNN/8.0.4.30-CUDA-11.1.1 and 1 dependency loaded.
/home/h3/medranos/vdftb20/lib/python3.6/site-packages/schnetpack/data/atoms.py:327: UserWarning: The given NumPy array is not writeable, and PyTorch does not support non-writeable tensors. This means you can write to the underlying (supposedly non-writeable) NumPy array using the tensor. You may want to copy the array to protect its data or make it writeable before converting it to a tensor. This type of warning will be suppressed for the rest of this program. (Triggered internally at  /pytorch/torch/csrc/utils/tensor_numpy.cpp:143.)
  properties[pname] = torch.FloatTensor(prop)
/home/h3/medranos/vdftb20/lib/python3.6/site-packages/ase/atoms.py:968: VisibleDeprecationWarning: Use get_global_number_of_atoms() instead
  np.VisibleDeprecationWarning)
/scratch/ws/1/medranos-TUDprojects/raghav/Prop_pred/bob_conv.py:244: UserWarning: nn.init.xavier_uniform is now deprecated in favor of nn.init.xavier_uniform_.
  torch.nn.init.xavier_uniform(m.weight)
Traceback (most recent call last):
  File "/scratch/ws/1/medranos-TUDprojects/raghav/Prop_pred/bob_conv.py", line 453, in <module>
    int(train_set[ii]), int(n_val), int(n_test), iX, iY, patience
  File "/scratch/ws/1/medranos-TUDprojects/raghav/Prop_pred/bob_conv.py", line 360, in fit_model_dense
    train_nn(train_loader, model, loss_fn, optimizer)
  File "/scratch/ws/1/medranos-TUDprojects/raghav/Prop_pred/bob_conv.py", line 301, in train_nn
    pred = model(X)
  File "/home/h3/medranos/vdftb20/lib/python3.6/site-packages/torch/nn/modules/module.py", line 889, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/home/h3/medranos/vdftb20/lib/python3.6/site-packages/torch/nn/parallel/data_parallel.py", line 167, in forward
    outputs = self.parallel_apply(replicas, inputs, kwargs)
  File "/home/h3/medranos/vdftb20/lib/python3.6/site-packages/torch/nn/parallel/data_parallel.py", line 177, in parallel_apply
    return parallel_apply(replicas, inputs, kwargs, self.device_ids[:len(replicas)])
  File "/home/h3/medranos/vdftb20/lib/python3.6/site-packages/torch/nn/parallel/parallel_apply.py", line 86, in parallel_apply
    output.reraise()
  File "/home/h3/medranos/vdftb20/lib/python3.6/site-packages/torch/_utils.py", line 429, in reraise
    raise self.exc_type(msg)
RuntimeError: Caught RuntimeError in replica 0 on device 0.
Original Traceback (most recent call last):
  File "/home/h3/medranos/vdftb20/lib/python3.6/site-packages/torch/nn/parallel/parallel_apply.py", line 61, in _worker
    output = module(*input, **kwargs)
  File "/home/h3/medranos/vdftb20/lib/python3.6/site-packages/torch/nn/modules/module.py", line 889, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/scratch/ws/1/medranos-TUDprojects/raghav/Prop_pred/bob_conv.py", line 282, in forward
    layer0 = self.lin0(input_vector)
  File "/home/h3/medranos/vdftb20/lib/python3.6/site-packages/torch/nn/modules/module.py", line 889, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/home/h3/medranos/vdftb20/lib/python3.6/site-packages/torch/nn/modules/linear.py", line 94, in forward
    return F.linear(input, self.weight, self.bias)
  File "/home/h3/medranos/vdftb20/lib/python3.6/site-packages/torch/nn/functional.py", line 1753, in linear
    return torch._C._nn.linear(input, weight, bias)
RuntimeError: mat1 dim 1 must match mat2 dim 0

